src/table/utils/__tests__/selections-util.spec.js

```diff
@@ -34,24 +31,20 @@ describe('selections-utils', () => {
       };
     });
 
-    afterEach(() => {
-      sinon.verifyAndRestore();
-      sinon.resetHistory();
-    });
+    afterEach(() => jest.clearAllMocks());
 
     it('should call api.on and api removeListener for all listeners', () => {
       addSelectionListeners({ api, selectionDispatch, setShouldRefocus, keyboard, tableWrapperRef })();
-      listenerNames.forEach((name) => {
-        expect(api.on).to.have.been.calledWith(name);
```

Comment:
Failed in switching to Jest 
Tried with like
`expect(api.on).toHaveBeenNthCalledWith(1, 'deactivated', resetSelections);`
Do not why it does not work

Reply:
Try `expect(api.on).toHaveBeenNthCalledWith(1, 'deactivated', expect.anything());`. If it works, put the loop back

Reply:
Nice one